app/views/admin/media/_media_table.html.erb

```diff
@@ -48,7 +48,14 @@
     <% @media.each do |medium| %>
     <tr>
       <td>
-        <a href="<%= url_for(medium.file) %>" rel="noopener" target="_blank"><%= medium.name %></a>
+        <% if medium.file.image? %>
+          <%= image_tag medium.file.variant(resize_to_limit: [100, 100]) %><br>
+        <% end %>
+        <a href="<%= url_for(medium.file) %>" rel="noopener" target="_blank"><%= medium.name %></a><br>
+        <span><%= number_to_human_size(medium.file.byte_size) %></span><br>
+        <% if medium.file.image? %>
```

Comment:
I would suggest to make sure that both width and height exist. Sometimes sidekiq takes time to calculate the metadata (and sometimes it is down :P)